src/utils/hooks/api/use-get-prices.tsx

```diff
@@ -65,11 +103,19 @@ type Prices = {
   [ticker: string]: Price;
 };
 
+type PricesSource = 'oracle' | 'api';
```

Comment:
IMO more elegant to properly type this via an enum like:

```suggestion
enum PricesSource {
  Coingecko,
  Oracle
}
```

Reply:
I will make I include these changes tomorrow. Thank you